modin/pandas/base.py

```diff
@@ -2791,6 +2791,23 @@ def sort_values(
              A sorted DataFrame.
         """
         axis = self._get_axis_number(axis)
+        inplace = validate_bool_kwarg(inplace, "inplace")
+        if axis == 0:
+            result = self._query_compiler.sort_rows_by_column_values(
+                by,
+                ascending=ascending,
+                na_position=na_position,
+                ignore_index=ignore_index,
+            )
+        else:
+            result = self._query_compiler.sort_columns_by_row_values(
+                by,
+                ascending=ascending,
+                na_position=na_position,
+                ignore_index=ignore_index,
+            )
+        return self._create_or_update_from_compiler(result, inplace)
```

Comment:
@devin-petersohn you need to remove code after `return` statement.

Reply:
Thanks @anmyachev, I have removed it.